semantic-java/src/Language/Java/Tags.hs

```diff
@@ -76,6 +80,24 @@ instance ToTags Java.MethodInvocation where
       Tags.yield (Tag name Call loc (Tags.firstLine src range) Nothing)
       gtags t
 
+instance ToTags Java.InterfaceDeclaration where
+  tags t@Java.InterfaceDeclaration
+    { ann = loc@Loc { byteRange = Range { start } }
+    , name = Java.Identifier { text = name }
+    , body = Java.InterfaceBody { ann = Loc Range { start = end } _ }
+    } = do
+      src <- ask @Source
+      Tags.yield (Tag name Interface loc (Tags.firstLine src (Range start end)) Nothing)
```

Comment:
Similar question about `Tags.firstLine` here, and if we could use the `byteRange` instead? I think that would help us stay consistent with how we record the byterange for tags instances in other langs (where we take the entire range of the subtree).

Reply:
Fixing!